lib/transform/propertySetup.js

```diff
@@ -29,7 +29,7 @@ function propertySetup(property, name, path) {
   if (!property && !_.isPlainObject(property))
     throw new Error('Property object must be an object');
   if (!name || !_.isString(name))
-    throw new Error('Name must be a string');
+    throw new Error('Name must be a string at path: ' + _.isArray(path) ? path.join('.') : path);
```

Comment:
In v4 we're aiming to ditch lodash and es6 helpers, you can use native `Array.isArray()` instead